### PR TITLE
Cob/sequential trx requests

### PR DESCRIPTION
**Pull request by coburncoburn**
.exec (My Trx Request) added to repl
sort trx requests in transaction pool

### CODE REVIEW

File: chains/scripts/add_users.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+async function feedAccount(n, r, accounts = [] ) {
+    const user = saddle.account
+    var accounts
+    var cash_nonce = n
+    if (r > 2000) return;
+
+    if (accounts.length == 0 || i == accounts.length) {
+        console.log(r)
+        accounts =  
+            await fetch(`https://api.compound.finance/api/v2/account?page_number=${r}&page_size=200`)
```
Comment (hayesgm) [inline chains/scripts/add_users.js:15]: Wouldn't it just have been easier to do `0x000000000...1`, `0x000000000...2` and so on?

File: chains/scripts/README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+usage: scripts to be run in the context of the repl
```
Comment (hayesgm) [inline chains/scripts/README.md:1]: I think we already have an option for this in the repl, if I recall it should be `yarn repl -s ./scripts/add_user.js`

**Review by hayesgm — APPROVED** [review]
Overall looks great.

File: chains/scripts/add_users.js (hunk 1 above)
Comment (coburncoburn) [reply chains/scripts/add_users.js:15]: yeah, maybe I'll change it up and do both so we get "realer" data and also more pathological data.

File: chains/scripts/README.md (hunk 2 above)
Comment (coburncoburn) [reply chains/scripts/README.md:1]: oh cool. is there one that keeps the shell open after running the script?

**Pull request merged**